packages/editor/src/components/collapsible-block-toolbar/style.scss

```diff
@@ -63,7 +65,7 @@
 		@include break-small() {
 			&:not(.is-horizontal) .block-editor-block-mover__move-button-container {
 				position: relative;
-				top: -10px;
+				top: - $grid-unit;
```

Comment:
Same here.

Reply:
Increasing this value will shift the toolbar when focusing on the block mover button on Windows OS. I should have left a more detailed comment. See [this comment](https://github.com/WordPress/gutenberg/pull/60729#issuecomment-2058230476) for details. If we can make the header height 64px in #60729, there may be no problem with applying `top: - $grid-unit;`.

https://github.com/WordPress/gutenberg/assets/54422211/ef0a1291-96f4-4fba-89e3-2318367256ed

Reply:
If we don't adapt it, it'll stay visually mis-aligned all the time. I suppose it's one + for the 64px header height.

Reply:
> If we don't adapt it, it'll stay visually mis-aligned all the time

Yes, I understand that too. This point has been discussed in #57444, where custom scrollbars were introduced (See [this comment](https://github.com/WordPress/gutenberg/pull/57444#issuecomment-1909695979)). Do you think it's okay to merge #60729 first? After that, we can check again to see if `top: - $grid-unit;` does not cause a shift.

Reply:
I'm ok taking out that `top: - $grid-unit;` adjustment for now, to not hold this up.